worker.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+
+const precise = require('precise'),
+  retsu = require('retsu'),
+  LRUCache = require('lru_cache').LRUCache,
+  Simple = require('simple-lru-cache'),
+  Fast = require('lru-fast').LRUCache,
+  QuickLRU = require('quick-lru'),
+  Modern = require('modern-lru'),
+  hyperlru = require('hyperlru'),
+  {LRUMap} = require('lru_map'),
+  MKC = require('mkc'),
+  hyperlruObject = hyperlru(require('hyperlru-object')),
+  hyperlruMap = hyperlru(require('hyperlru-map')),
+  caches = {
+    'lru-cache': () => require('lru-cache'),
+    'lru-fast': n => new Fast(n),
+    'js-lru': n => new LRUMap(n),
+    'modern-lru': n => new Modern(n),
+    'quick-lru': maxSize => new QuickLRU({maxSize}),
+    'secondary-cache': require('secondary-cache'),
+    'simple-lru-cache': maxSize => new Simple({maxSize}),
+    'tiny-lru': require('tiny-lru'),
+    hashlru: require('hashlru'),
+    'hyperlru-object': max => hyperlruObject({max}),
+    'hyperlru-map': max => hyperlruMap({max}),
+    lru_cache: n => new LRUCache(n),
+    lru: require('lru'),
+    mkc: max => new MKC({max})
+  },
+  num = 2e5,
+  evicts = num * 2,
+  times = 5,
+  x = 1e6,
```

Comment:
what is that?

Reply:
i don't understand the question, it's a number; it's used to turn nanoseconds into milliseconds.